--- ORIGINAL
+++ REPUBLICADO
@@ -5,4 +5,4 @@
  |  |  | 144967729.68
- | Valor do OPEX: |  | 1040707659.8720034 | (considerando todo o período de concessão)
- | Valor do Contrato: |  | 1743398724.0872536 | (soma das receitas ordinárias previstas para todo o período da concessão)
+ | Valor do OPEX: |  | 1040707059.9546887 | (considerando todo o período de concessão)
+ | Valor do Contrato: |  | 1742903402.894245 | (soma das receitas ordinárias previstas para todo o período da concessão)
  | WACC: |  | 0.11686498859823251 | a.a. real
@@ -12,8 +12,8 @@
  | Payback simples |  | 10 | anos
- | Tarifa-Base Referencial (R$/m³) |  | 2.8567488023289074
- | Tarifa Básica dos Entes Públicos (R$ por tonelada) |  | 223.6683778869185
- | Valor da tarifa por economia  |  | 413.29891997533355 | ano
- | (no ano 1) - média |  | 34.44157666461113 | mês
- | Valor da tarifa per capita |  | 181.45425367422848 | ano
- | (no ano 1) - média |  | 15.121187806185707 | mês
+ | Tarifa-Base Referencial (R$/m³) |  | 2.855879456931295
+ | Tarifa Básica dos Entes Públicos (R$ por tonelada) |  | 223.66777191751288
+ | Valor da tarifa por economia  |  | 413.17314779906593 | ano
+ | (no ano 1) - média |  | 34.43109564992216 | mês
+ | Valor da tarifa per capita |  | 181.3990348113798 | ano
+ | (no ano 1) - média |  | 15.11658623428165 | mês
 [18 linhas]