--- ORIGINAL
+++ REPUBLICADO
@@ -32,7 +32,7 @@
  |  | Agente Financeiro | 2898000
- |  | Seguros e Garantias | 3781333.9858096046
+ |  | Seguros e Garantias | 3780734.0684948284
  |  | Despesas Administrativas e de Pessoal SPE | 65719940.4
  |  | Inclusão social e emancipação econômica de catadores | 43426342.42700001
- |  | Consórcio Gestão Contratual | 10800000
- |  | Total | 1033507659.8720036
+ |  | Consórcio Gestão Contratual | 18000000
+ |  | Total | 1040707059.9546888
  |  | OPEX - Considerando todo o período de concessão | Total
@@ -45,7 +45,7 @@
  |  | Despesas com órgão regulador | 26361232.270298854
- |  | Despesas gestão e cobrança tarifária | 157530874.4602671
+ |  | Despesas gestão e cobrança tarifária | 157530274.54295233
  |  | Despesas Administrativas e de Pessoal SPE | 65719940.4
  |  | Inclusão social de catadores | 43426342.42700001
- |  | Consórcio Gestão Contratual | 10800000
- |  | Total | 1033507659.8720034
+ |  | Consórcio Gestão Contratual | 18000000
+ |  | Total | 1040707059.9546887
  | 4. | Tabela - OPEX Detalhado
@@ -66,3 +66,3 @@
  |  | Agente Financeiro | 2898000
- |  | Seguros e Garantias | 3781333.9858096046
+ |  | Seguros e Garantias | 3780734.0684948284
  |  | Despesas Administrativas e de Pessoal SPE | 65719940.4
@@ -70,3 +70,3 @@
  |  | Inclusão social e emancipação econômica de catadores | 43426342.42700001
- |  | Total | 1040707659.8720036
+ |  | Total | 1040707059.9546888
  | 5. | Tabela  - Seguros e Garantias